src/romloader.cpp

```diff
@@ -15,7 +15,7 @@ bool RomLoader::load(std::string const & romName)
     if (fd > 0) {
         _data.fill(0);
         uint8_t buff[512];
-        int readCount = 0;
+        int readCount = 0x0100;
```

Comment:
is this correct ? if so maybe use a variable to let us know what is it. (don't use magic numbers)

Reply:
Can't find why I did this, putting it on toto